crates/aptos/e2e/cases/account.py

```diff
@@ -80,3 +81,28 @@ def test_account_lookup_address(run_helper: RunHelper, test_name=None):
         raise TestError(
             f"lookup-address result does not match {run_helper.get_account_info().account_address}"
         )
+
+
+@test_case
+def test_account_rotate_key(run_helper: RunHelper, test_name=None):
```

Comment:
I'd like to see a bit more to this test. Just because the CLI says it was successful doesn't mean it was actually successful. Could you make a request to the local testnet to verify that the key rotation actually worked?